android_app/app/src/main/java/com/health/openscale/core/bluetooth/BluetoothStandardWeightProfile.java

```diff
@@ -823,29 +825,59 @@ protected void chooseExistingScaleUser(Vector<ScaleUser> userList) {
         chooseScaleUserUi(choices);
     }
 
-    protected String getInitials(String fullName) {
-        if (fullName == null || fullName.isEmpty() || fullName.chars().allMatch(Character::isWhitespace)) {
-            return getDefaultInitials();
-        }
-        return buildInitialsStringFrom(fullName).toUpperCase();
+    protected String getInitials(final String fullName, final int size) {
+        if (0 == size)
+            return StringUtils.EMPTY;
+
+        if (StringUtils.isAnyBlank(fullName))
+            return getDefaultInitials(size);
```

Comment:
Any reason to use `isAnyBlank()` instead of `isBlank()`?
```suggestion
        if (StringUtils.isBlank(fullName))
            return getDefaultInitials(size);
```

Reply:
You're right. Great spot. Modified accordingly